### DISCUSSION TITLE
Looking for Collaborators / Help

### DISCUSSION THREAD

**Discussion by mlmbusiness83** [community/community#133139] [New to GitHub]
I have a AI ecosystem i built name Apex-Echo-AI. I integrated into a Multi-Vendor MarketPlace.  I will give away a free copy of Apex-Echo-AI for your help and collaboration with testing and help launching to the hosting provider.  If you would like to know more, please reach out to me. This is an urgent matter.

**Comment by coderAdityasen** [comment]
hey, is there anything i can help you with in a particular topic and i am interested in collaboration with you, you can reach out to me on 
discord :- @adittyaassen
mail :- adityasen561@gmail.com

**Comment by ghostinhershell** [comment]
Hi @mlmbusiness83, we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

**Comment by Sarun2515** [comment]
นักพัฒนาหลายล้านคนทั่วโลกโฮสต์โครงการหลายล้านโครงการทั้งแบบโอเพ่นซอร์สและแบบปิดบน GitHub เราโชคดีที่ได้มีส่วนช่วยในการส่งเสริมการทำงานร่วมกันในชุมชนนักพัฒนาทุกวัน ซึ่งเป็นความรับผิดชอบที่เราไม่มองข้าม เราทุกคนมีโอกาสอันน่าตื่นเต้นที่จะสร้างชุมชนนี้ให้น่าภาคภูมิใจร่วมกัน

**Comment by Sarun2515** [comment]
ด้วยจรรยาบรรณนี้ เราหวังว่าเราจะช่วยให้คุณเข้าใจว่าควรทำงานร่วมกันใน GitHub Community อย่างไร คุณคาดหวังอะไรได้บ้างจากผู้ดูแลระบบ และประเภทของการดำเนินการหรือเนื้อหาใดที่อาจส่งผลให้คุณถูกระงับการเข้าร่วมชุมชนชั่วคราวหรือถาวร เราจะตรวจสอบรายงานการละเมิดใดๆ และอาจควบคุมเนื้อหาสาธารณะภายใน GitHub Community ที่เราพิจารณาว่าละเมิดข้อกำหนดในการให้บริการของ GitHub หรือจรรยาบรรณนี้

**Comment by Sarun2515** [comment]
ฐานผู้ใช้ที่หลากหลายของเรามีมุมมอง แนวคิด และประสบการณ์ที่แตกต่างกัน นับตั้งแต่ผู้คนที่สร้างโปรเจ็กต์ "Hello World" แรกเมื่อสัปดาห์ที่แล้ว ไปจนถึงนักพัฒนาซอฟต์แวร์ที่มีชื่อเสียงที่สุดในโลก เรามุ่งมั่นที่จะทำให้ GitHub เป็นสภาพแวดล้อมที่เปิดรับเสียงและมุมมองที่หลากหลายจากชุมชนของเรา ขณะเดียวกันก็รักษาสถานที่ที่ปลอดภัยสำหรับนักพัฒนาในการทำงานให้ดีที่สุด
